app/validators/profile_validator.rb

```diff
@@ -45,9 +42,8 @@ def check_box_valid?(_record, _attribute)
     true # checkboxes are always valid
   end
 
-  def color_field_valid?(record, attribute)
-    hex_value = record.public_send(attribute)
-    hex_value.nil? || hex_value.match?(HEX_COLOR_REGEXP)
+  def color_field_valid?(_record, _attribute)
```

Comment:
We don't have any color profile fields right now since they all moved to `Users::Setting`. Eventually, we remove color fields from profiles altogether or extract the hex color validator into something more reusable, until then I need this dummy method (it seemed less bad than adding another guard clause to `validate`). Once I have clarity on the future of color pickers as possible input types for profile fields I'll do a follow-up.

Reply:
Can you transform this note into an inline comment? I'm sure people will wonder why that is there

Reply:
I already have a follow-up PR (https://github.com/forem/forem/pull/14239) that will remove color fields completely, which also gets rid of this dummy method. The only reason I want to merge this first is that the DUS gets scheduled for ~10 minutes after deploy, so we need `color_field` to still be around for a bit.